phpunit.xml

```diff
@@ -5,13 +5,6 @@
          cacheResult="false"
          colors="true"
 >
-    <coverage>
-        <report>
-            <clover outputFile="build/logs/clover.xml" />
-            <html outputDirectory="build/logs/coverage" />
-            <text outputFile="build/logs/coverage.txt" />
-        </report>
-    </coverage>
```

Comment:
I've recently started to notice that explicitly declared “coverage” parameters in the `phpunit.xml` file can show this notification.